src/generator/qbe.rs

```diff
@@ -135,8 +135,8 @@ impl QbeGenerator {
             None
         };
 
-        let mut qfunc = QbeFunction {
-            exported: true,
+        let mut qfunc = qbe::Function {
+            linkage: qbe::Linkage::public(),
             name: func.name.clone(),
             arguments,
             return_ty,
```

Comment:
This should use `qbe::Module`

Reply:
I might change the API of the crate a bit. Instead of

```rs
module.add_type(name, offset, items);
```

it would be nicer to leave the construction to the type itself:

```rs
let typedef = qbe::TypeDef::new(name, offset, items);
module.add_type(typedef);
```

That way we can consolidate the functions better:

```rs
let module = qbe::Module::new();
let structure = self.generate_struct(def)?;
module.add_type(structure);
```